README.md

```diff
@@ -30,6 +30,10 @@ This SDK version is compatible with the Stark Infra API v2.
         - [Stock](#query-issuingstocks): View your current stock of a certain IssuingDesign linked to an Embosser on the workspace
         - [Restock](#create-issuingrestocks): Create restock orders of a specific IssuingStock object
         - [EmbossingRequest](#create-issuingembossingrequests): Create embossing requests
+        - [TokenRequest](#create-an-issuingtokenrequest): Create token requests
+        - [TokenAuthorization](#process-issuingtokenauthorizations): process token authorizations
+        - [TokenActivation](#process-issuingtokenactivation): process token activation 
```

Comment:
- [TokenActivation](#process-issuingtokenactivation): Get notified on how to inform the activation code to the holder

Reply:
alinhar